### DISCUSSION TITLE
docs/syntax

### DISCUSSION THREAD

**Discussion by giscus** [flowershow/flowershow#646] [Announcements]
# docs/syntax

Flowershow was designed with Obsidian users in mind, and so, it aims to fully support Obsidian syntax, including CommonMark, GitHub Flavoured Markdown and Obsidian extensions, like Wiki links.Here are...

https://flowershow.app/docs/syntax

**Comment by Iwuaa** [comment]
Seems like some of these don't work, looking how it renders here: 
- highlighting
- excalidraw sketches
- comments
- ellipses
- inline code - is that how it's supposed to look?

**Reply by rufuspollock** [reply]
I think these are working looking at the page above. Which are not working for you?

**Comment by Iwuaa** [comment]
Your [Assets page](https://flowershow.app/assets) is not working - I get 404

**Reply by rufuspollock** [reply]
Yes, the directory gives 404 but i think all the assets in the folder are working.
